include/libdnf/rpm/nevra.hpp

```diff
@@ -30,7 +31,12 @@ struct Nevra {
 public:
     enum class Form { NEVRA = 1, NEVR = 2, NEV = 3, NA = 4, NAME = 5 };
 
+    /// The order is according to propability what subject Form could be the most
```

Comment:
Maybe replace the comment with following?:
The forms and their order determine pkg_spec matching